recipes/rtmidi/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+import os
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class RtMidiConan(ConanFile):
+    name = "rtmidi"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://www.music.mcgill.ca/~gary/rtmidi/"
+    description = "Realtime MIDI input/output"
+    topics = ("midi")
+    license = "MIT+send-patches-upstream"
```

Comment:
🤔 At first this seemed odd but I have no clue what would be better...

Anyone else have ideas?

Reply:
It's not MIT, but it's an official license. However, there are more projects using the same. The sentence "send patches upstream" is easily found when search for this license, so it sounds good. What could be done, submitting it as official open source license.